std/encoding.d

```diff
@@ -1660,8 +1659,7 @@ immutable(E)[] sanitize(E)(immutable(E)[] s)
     t = s[n..$];
     while (t.length != 0)
     {
-        dchar c = EncoderInstance!(E).safeDecode(t);
-        assert(c == INVALID_SEQUENCE);
+        assert(EncoderInstance!(E).safeDecode(t) == INVALID_SEQUENCE);
```

Comment:
No, because this will get compiled out in release mode! you have to keep the verbose version.